src/sarif_converter.ts

```diff
@@ -60,7 +60,7 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
       id: suggestionId,
       name: suggestion.rule,
       shortDescription: {
-        text: suggestion.message,
+        text: suggestion.title || suggestion.message,
```

Comment:
🤔 
`suggestion.message` should not be in `rule` - as it can contain values that are specific to a specific instance of a suggestion. So it can make us show unrelated symbol names in the title of of an issue of the same rule.
In addition, `suggestion.message` might be too long sometimes for a title.

To temporarily support rules without `title` maybe do
```
text: suggestion.title  || suggestion.rule
```
instead?

Reply:
ok

Reply:
done